packages/uniforms-bridge-json-schema/src/JSONSchemaBridge.ts

```diff
@@ -111,32 +131,43 @@ export default class JSONSchemaBridge extends Bridge {
     const rootName = joinName(nameParts.slice(0, -1));
     const baseName = nameParts[nameParts.length - 1];
     const scopedError = details.find(error => {
-      const path = pathToName(error.instancePath ?? error.dataPath);
+      const rawPath = error.instancePath ?? error.dataPath;
+      const path = rawPath ? pathToName(rawPath) : '';
       return (
         unescapedName === path ||
-        (rootName === path && baseName === error.params.missingProperty)
+        (rootName === path &&
+          error.params &&
+          baseName === error.params.missingProperty)
       );
     });
 
     return scopedError || null;
   }
 
-  // TODO: Get rid of this `any`.
-  getErrorMessage(name: string, error: any) {
+  getErrorMessage(name: string, error: unknown) {
     const scopedError = this.getError(name, error);
     return scopedError?.message || '';
   }
 
-  // TODO: Get rid of this `any`.
-  getErrorMessages(error: any) {
+  getErrorMessages(error: unknown) {
     if (!error) {
       return [];
     }
 
-    const { details } = error;
-    return Array.isArray(details)
-      ? details.map(error => error.message)
-      : [error.message || error];
+    if (error instanceof Error) {
+      return [error.message];
+    }
+
+    if (isValidatorResult(error)) {
+      const { details } = error;
+      return details.map(error => error.message || '');
+    }
+
+    if (typeof error === 'object') {
+      return [];
+    }
+
+    return ['' + error];
```

Comment:
Won't you prefer to convert it to string specifically, using `.toString()`? I know it's still JS, but it seems a little bit 'hacky' to me.

Reply:
I don't like `.toString()` if I somehow missed checking for `null` or `undefined`.

Reply:
But both `undefined` and `null` got eliminated in `if (!error)` above.

Reply:
![image](https://user-images.githubusercontent.com/13087338/216573762-b229b041-c08b-4290-b9d7-ccfa00e8e51d.png)
🙉